### DISCUSSION TITLE
rf recv does not appear in the configuration menu (tasmota 9.3.1.2)(SOLVED)

### DISCUSSION THREAD

**Discussion by lisergio** [arendst/Tasmota#11631] [Q&A] [answered]
I have programmed an esp8266 03 using the latest available version of tasmota, and when assigning the gpios, the RF Recv option does not appear in the options.
Do you have to compile it by activating any option?

thanks

**Comment by sfromis** [comment, marked as answer]
Yes, compile is needed. See the comments in https://tasmota.github.io/docs/Components/

**Reply by lisergio** [reply]
thanks..
I have read it, but I don't understand it ...
I understand that since there are more options, it is configured in arrays,
but I don't understand how to compile so that the other configuration options appear
in the code I do not find that I have to change.

thanks again

**Reply by rapi3** [reply]
try tasmocompiler:
https://github.com/benzino77/tasmocompiler

**Reply by sfromis** [reply]
What you write does not make me believe that you actually found the part about RFrecv in that page. There is nothing about configuring in arrays, you have to add configuration lines.

For fuller info on compiles, see https://tasmota.github.io/docs/Compile-your-build/

**Reply by lisergio** [reply]
Thank you..
I'm going to read the documentation and have to try

**Reply by lisergio** [reply]
I have read the document, and I have figured out how to do it ...
You have to edit the file "my_user_config.h" and uncomment the option you want to activate ...
in my case it is on line 824
"// define USE_RC_SWITCH"
uncomment it so that it stays that way ...
"define USE_RC_SWITCH"

I have used the Arduino IDE to do it ..

Thanks for the help

**Reply by sfromis** [reply]
Two points to add. The recommended procedure is to not modify `my_user_config.h` but instead make the amendments in `user_config_override.h`. This also simplifies repeating the changes for future version. And the old Arduino IDE is no longer supported for building, many compiles (but not all) will fail unless using the recommended, from the link I already posted.

**Reply by lisergio** [reply]
ok!
thanks!
